### PR TITLE
Add translation platform details in about page

### PR DESCRIPTION
**Pull request by Altonss**
I hope that this PR fixes issue #652 :)

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -257,4 +257,5 @@
     <string name="report_error">Report Error</string>
     <string name="action_show_details">Show details</string>
     <string name="action_hide_details">Hide details</string>
+    <string name="translate_platform">on hosted Weblate</string>
```
Comment (TheLastProject) [inline app/src/main/res/values/strings.xml:260]: Let's just use "on Weblate", feels cleaner :)

**Review by TheLastProject — CHANGES_REQUESTED** [review]
Looks good, just one small thing

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Altonss) [reply app/src/main/res/values/strings.xml:260]: Okay I can change that :)

**Review by Altonss — COMMENTED** [review]
I have now changed to 'on Weblate'

**Comment by TheLastProject** [comment]
Thanks!

**Pull request merged**